Jenkinsfile

```diff
@@ -34,6 +36,21 @@ pipeline {
         step([$class: 'HockeyappRecorder', applications: [[apiToken: env.HOCKEYAPP_API_KEY, downloadAllowed: true, filePath: 'sample/build/outputs/apk/sample-hockey-debug.apk', mandatory: false, notifyTeam: false, releaseNotesMethod: [$class: 'NoReleaseNotes'], uploadMethod: [$class: 'VersionCreation', appId: env.HOCKEYAPP_ID]]], debugMode: false, failGracefully: false])
       }
     }
+    stage('Release SDK') {
+      when {
+        branch 'master'
+      }
+      steps {
+        script {
+             def bintrayCredentials = input message: 'Enter your Bintray credentials', parameters: [string(defaultValue: '', description: 'Bintray username', name: 'BINTRAY_USERNAME'), password(defaultValue: '', description: 'Bintray api key', name: 'BINTRAY_KEY')]
```

Comment:
Is this the thing where Jenkins will show a prompt and you manually have to login and type your credentials and confirm the release? And it would wait for a day or some for you to do that?

Reply:
yes